### ISSUE TITLE
Deprecated Methods

### ISSUE THREAD

**Issue by giovannifrisina** [vdemydiuk/mtapi#141]
Transferring the solution on VS2017, I've had some issues about obsolete methods. I think the porting tried to convert the functions without result. How can I resolve the issue? For example:

private void OrderClosePrice()
{
            var result = _apiClient.OrderClosePrice();
            textBoxOrderPrice.Text = result.ToString(CultureInfo.CurrentCulture);
            PrintLog($"OrderClosePrice result: {result}");
 }

[Obsolete("OrderClosePrice is deprecated, please use GetOrder instead.")]
public double OrderClosePrice()
{
            return SendCommand<double>(MtCommandType.OrderClosePrice, null);
}

Thanks

**Comment by KptKuck** [comment]
This is not a error. It's just a compiler warning. The developer has marked these features as obsolete. You can ignore or disable the warning.

**Comment by vdemydiuk** [comment]
Yes, some functions are marked as obsolete and they have another implementation for using in the applications.
I think there is a time to remove them from the API after migration the project to VS2017.
